example/scripts/deployMultichain.ts

```diff
@@ -0,0 +1,35 @@
+import { multichain } from "hardhat";
+import { NetworkArguments } from "@chainsafe/hardhat-plugin-multichain-deploy";
+
+async function main(): Promise<void> {
+  const currentTimestampInSeconds = Math.round(Date.now() / 1000);
+  const unlockTime = BigInt(currentTimestampInSeconds + 60);
+
+  const networkArguments = ["sepolia", "mumbai", "holesky"].reduce(
```

Comment:
maybe it would be clearer if you manually set those for each network instead of reducing them. It's a sample so more declarative code is better than concise code imho

Reply:
can do that!